docs/application/dotnet/guides/nui/animation-types.md

```diff
@@ -2,15 +2,15 @@
 
 ## Path Animation
 
-The `Path` class can be used to animate the position and orientation of actors. (a three-dimentional parametric curve)
+The `Path` class is three-dimensional parametric curve that is used to animate the position and orientation of actors.
 
 The `Path` consists of several points and each adjacent pair of points becomes a segment of the path. Each segment can be curved by adding control points to it.
 
 In the following example, the P1, P2 and P3 form the path where the blue circle travels. The C1 and C2 are the control points for a first segment `[P1-P2]`, where C3 and C4 are for a second segment `[P2-P3]`. The control points express the curvature of the path.
 
 <div style="text-align:center;width:100%;"><img src="./media/path.svg" /></div>
 
-The code would look like:
+To generate `Path' object, use the following code:
```

Comment:
There is a just a missing article 'the', please make the small change:

To generate the `Path' object, use the following code: